src/components/GetRooms.vue

```diff
@@ -11,15 +11,15 @@
 
 <script>
 import firebase from "firebase/compat/app";
-// import * as firebaseui from "firebaseui";
 import "firebase/auth";
 import "firebase/database";
 import {
   getFirestore,
   collection,
   getDocs,
   addDoc,
-} from "firebase/firestore/lite";
+  onSnapshot
+} from "firebase/firestore";
```

Comment:
I think the `/lite` version of didn't have the `onSnapshot` stuff for live updates.